### PR TITLE
docs: official docs for saved replies

### PR DESCRIPTION
**Pull request by eddiejaoude**
Unfortunately Sema is no longer available, but we can still use the templates with GitHub "Saved Replies"

### CODE REVIEW

**Review by Panquesito7 — DISMISSED** [review]
Looks good to me! :)

File: docs/snippets/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### Pull Request Saved Replies
```
Comment (schmelto) [inline docs/snippets/README.md:1]:
```suggestion
# Pull Request Saved Replies
```

Linter issue this should be a top level heading

**Review by Notchoaveragejoe — DISMISSED** [review]
Should be top level heading.

**Review by schmelto — APPROVED** [review]
## Looking good to me

Your suggested changes looking good to me.

---
**Summary:** :ok_hand: This code looks good

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**